nengo_viz/static/viz.js

```diff
@@ -4,10 +4,21 @@ var VIZ = {};
 VIZ.max_zindex = 0;
 
 /**
+ * Helper function to clip a number, keeping it between two values.
+ */
+VIZ.clip = function(x, low, high) {
+    if (x < low)
+        x = low;
+    if (x > high)
+        x = high;
+    return x;
+}
+
+/**
  * Helper function to set the transform of an element.
  */
 VIZ.set_transform = function(element, x, y) {
-    element.style.webkitTransform = 
+    element.style.webkitTransform =
```

Comment:
Yeah, I had done that before we had the discussion yesterday. When we're looking at merging, I (or whoever is merging) can go through and take out all the whitespace stuff.